posthog-react-native/babel.config.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  presets: ['module:metro-react-native-babel-preset', '@babel/env', '@babel/preset-typescript'],
+  presets: ['module:metro-react-native-babel-preset', '@babel/env'],
```

Comment:
is this intentional? this is still being set in the rollup.config.js, though

Reply:
I don't think we need it as part of the Jest tests which is the only place this config is being used. As you said it is still included as a babel preset in the rollup config